src/Cooked/MockChain/Testing.hs

```diff
@@ -17,10 +17,11 @@ import Ledger qualified
 import Test.QuickCheck qualified as QC
 import Test.Tasty.HUnit qualified as HU
 
--- | This module provides a common interface for HUnit and QuickCheck tests. We
--- do so by abstracting uses of 'HU.Assertion' and 'QC.Property' for @(IsProp
--- prop) => prop@, then provide instances for both @HU.Asserton@ and
--- @QC.Property@.
+-- * Common interface between HUnit and QuickCheck
+
+-- | 'IsProp' is a common interface for HUnit and QuickCheck tests. It abstracts
+-- uses of 'HU.Assertion' and 'QC.Property' for @(IsProp prop) => prop@, then
+-- provide instances for both @HU.Asserton@ and @QC.Property@.
```

Comment:
I think we will need, after this PR, to study whether we still use it.

Reply:
Yes, we have not used it for a while. Though I don't think it hurts being there in case we need it again at some point. That's why I left it.